neurom/features/neuronfunc.py

```diff
@@ -216,7 +216,7 @@ def sholl_crossings(neurites, center, radii):
     """Calculate crossings of neurites.
```

Comment:
this is not a feature with @feature

Reply:
@asanin-epfl , there is a small bug here

Reply:
> this is not a feature with @feature

Why isn't this a feature?

Reply:
it does not have the correc arguments, and is only used for the real feature sholl_frequency below. It crashed neurom for me saying missing values for center and radii arguments. But weird it didn't crash before. We need to investigate that

Reply:
it is not in the list in neuronfeatures that used to be here: https://github.com/BlueBrain/NeuroM/commit/ed3d9009e83cd3a742d86c82cfe23b905259e66c
In the test of the get function, it tests various features, but not this one (and not all possible features using `from neurom.apps.morph_stats import full_config` 
So I think this one could get pass undetected.